routes/articlesRoute.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by camel on 2016-01-06.
+ */
+
+module.exports = function (app) {
+    var articles = require('./../apiMethods/articlesMethods');
+    app.get('/articles/:id', articles.getSingleArticle);
+    app.get('/articles', articles.getAllArticle);
+    app.post('/articles', articles.addNewArticle);
+    app.put('/articles/:id', articles.updateArticle);
+    app.get('/articles/offset/:start/count/:amount/sort/:orderType', articles.getLimitedArticle);
```

Comment:
Moze tak to mozna rozwiazac ?
app.get('/articles/offset/:start/count/:amount/sort/:orderType', articles.getLimitedArticle);
app.get('/articles/sort/:orderType', articles.getLimitedArticle); i w getLimitedArticle obslugujesz ze jak nie ma start i amount to tylko sortuj, tak chyba bedzie ok.